mscore/continuouspanel.cpp

```diff
@@ -158,9 +158,20 @@ void ContinuousPanel::paint(const QRect&, QPainter& painter)
                   Staff* currentStaff = _score->staff(e->staffIdx());
                   Segment* parent = _score->tick2segment(Fraction::fromTicks(tick));
 
+                  // Find maximum width for the current Clef
+                  Clef* newClef = new Clef(_score);
+                  ClefType currentClef = currentStaff->clef(Fraction::fromTicks(tick));
+                  newClef->setClefType(currentClef);
+                  newClef->setParent(parent);
+                  newClef->setTrack(e->track());
+                  newClef->layout();
+                  if (newClef->width() > _widthClef)
+                        _widthClef = newClef->width();
+
                   // Find maximum width for the staff name
+                  qreal _nameWidth = 0;
                   QList<StaffName>& staffNamesLong = currentStaff->part()->instrument(Fraction::fromTicks(tick))->longNames();
-                  QString staffName = staffNamesLong.isEmpty() ? " " : staffNamesLong[0].name();
+                  QString staffName = staffNamesLong.isEmpty() ? "" : staffNamesLong[0].name();
```

Comment:
an empty string is different from one containing a space, is this really intended here?